app/html/caller_profile.html

```diff
@@ -194,19 +194,23 @@ <h1>User profile</h1>
         <div id="user_profile_section">
           <img width="200" src="/imgs/saveme.jpg" alt="">
           <p>First Name: </p>
-          <div id="detail_user_firstN">bank</div>
+          <div class="edit" id="detail_user_firstN" >bank</div>
           <p>Last Name: </p>
-          <div id="detail_user_lastN">name</div>
+          <div class="edit" id="detail_user_lastN" >name</div>
           <p>Eamil: </p>
-          <div id="detail_user_email">email</div>
+          <div class="edit" id="detail_user_email" >email</div>
```

Comment:
Don't need any changes here, just thought you could use this info. Right now the backend doesn't allow non-admin users to change their email and it won't show any errors if you try to submit an updated email. So if changes are made here they will not be reflected in the DB and no error will show. This is fine for now but the more people that know about it the better :)

Reply:
I was not sure about letting users change their email address. Yeah, I can make email to be disabled for changing.